test/time_feature/test_features.py

```diff
@@ -80,16 +82,21 @@
     ],
 )
 def test_feature_normalized_bounds(
-    feature: TimeFeature, index: pd.PeriodIndex
+    feature: TimeFeature, pd_index: pd.PeriodIndex
 ):
-    values = feature(index)
-    assert isinstance(values, np.ndarray)
-    for v in values:
-        assert -0.5 <= v <= 0.5
+    zb_index = zb.periods(pd_index[0], pd_index.freqstr, len(pd_index))
```

Comment:
So these tests now only check that the time features work with the zebras index, should we keep testing on the original pandas index?

Reply:
No? The below loop checks both pandas and zebras.

Reply:
woops, missed it